redis/src/test/scala/zio/redis/KeysSpec.scala

```diff
@@ -112,6 +125,50 @@ trait KeysSpec extends BaseSpec {
           restored <- get(key)
         } yield assert(restore)(isRight) && assert(restored)(isSome(equalTo(value)))
       },
+      suite("migrate")(
+        testM("migrate key to another redis server (copy and replace)") {
+          for {
+            key       <- uuid
+            value     <- uuid
+            _         <- set(key, value)
+            response  <- migrate("redis2",
+                                  6379,
+                                  key,
+                                  0L,
+                                  JavaDuration.ofMillis(5000),
+                                  copy = Option(Copy),
+                                  replace = Option(Replace),
+                                  keys = None)
+            out       =  ZIO.accessM[RedisExecutorWrapper](_.re.execute(Input.StringInput.encode("GET") ++ Input.StringInput.encode(key)))
```

Comment:
One idea of what might work here is an implicit conversion from the wrapper to a RedisExecutor.Service

Reply:
Maybe it makes sense to use `get(key).provide(secondExecutor)`. Alternatively, you could group executors in the same fashion it's done on `TestClock`, so that the dependency of this test is `RedisExecutor with SecondRedisExecutor`.

Reply:
Yeah provideLayer works